lib/wiki_api.rb

```diff
@@ -139,4 +150,8 @@ def typical_errors
      Faraday::ConnectionFailed,
      MediawikiApi::HttpError]
   end
+
+  def mediawiki_request_url
+    @mediawiki.instance_variable_get(:@conn).instance_variable_get(:@url_prefix).to_s
```

Comment:
Is this the full URL, or just the domain?

I guess that the `api_url` is the same domain plus some extra parts of the path?

In this case, we're reaching rather far into the MediawikiApi gem to try to find a URL, when we never dealt with a URL directly in the first place, instead leaving the construction of a URL from the query params completely up to gem. We should probably instead save the same set of data that we currently ship off to Sentry: the action, query and api_url.

Probably that means adding another column to error record table, for miscellaneous details.

Reply:
In this case, I actually have to verify whether or not the full URL is always there or not. During development when I raise the error after `@mediawiki.send(action, query)`, I get the full url correctly. But that raising of error is done after the request is correctly completed. On the other hand, the testing method(which uses stubbing and raises errors) gives only the domain. I actually have to see what actually happens in the real update scenario.

Reply:
Okay, I will add a `miscellaneous` column and save the details there.